core/common/byte_stream.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common/byte_stream.hpp"
+
+OUTCOME_CPP_DEFINE_CATEGORY(kagome::common, ByteStream::AdvanceErrc, e) {
+  using kagome::common::ByteStream;
+  switch (e) {
+    case ByteStream::AdvanceErrc::OUT_OF_BOUNDARIES:
```

Comment:
So did we finally decide to name constants LIKE_THIS rather than kLikeThis?

Reply:
This is not constant. Enumeration can be named using either of style: https://google.github.io/styleguide/cppguide.html#Enumerator_Names